core/src/main/kotlin/fr/sncf/osrd/api/pathfinding/PathfindingBlocksEndpoint.kt

```diff
@@ -224,16 +224,55 @@ fun runPathfinding(
     }
     val constraints = initConstraints(infra, rollingStocks!!)
 
-    // The heuristic is temporarily disabled, its implementation
-    // is currently too slow and the requests are actually slower.
-    // See https://github.com/OpenRailAssociation/osrd/issues/7200
-    val nullHeuristic: AStarHeuristicId<Block> = AStarHeuristic() { _, _ -> 0.0 }
-    val remainingDistanceEstimators = List(waypoints.size - 1) { nullHeuristic }
+    val remainingDistanceEstimators = makeHeuristics(infra, waypoints, 320.0 / 3.6)
```

Comment:
A bit off putting to see `320.0/3.6` coming out of nowhere. Make it into a variable? final static constant if you feel like it might be used elsewhere, if not at the minimum put it into a value just before method call?

Reply:
Ideally we'd add a parameter to the pathfinding requests, I'll open an issue and add a comment

Reply:
https://github.com/OpenRailAssociation/osrd/issues/8154

Comment added in the code

Reply:
I'd really like it to become a (global) constant in the meantime since its used in multiple places